docs-source/content/samples/simple/domains/model-in-image/initial.md

```diff
@@ -262,6 +262,7 @@ If you don't see the `weblogic-deploy.zip` file, then you missed a step in the [
 Now, you use the Image Tool to create an image named `model-in-image:WLS-v1` that's layered on a base WebLogic image. You've already set up this tool during the prerequisite steps.
 
 Run the following commands to create the model image and verify that it worked:
+Note: If you are creating a WebLogic model-in-image, add --chown oracle:root to imagetool.sh command. When creating FMW Infrastructure images, do not use the --chown option.
```

Comment:
1 - Since WLS is the most common case by far, we should just default to giving the 'correct' command.  
2 - Also, in the MII sample we simply use 'JRF' to indicated FMW Infrastructure.
3 - There are other location in MII sample doc and user doc that mention image tool.

So how about:

A - Add --chown oracle:root to the command line.

B - Replace the Note with:
{{% notice note %}}
If you are taking the JRF path through the sample, then  remove `--chown oracle:root` from the `imagetool.sh` command below. {{% /notice %}}

C - Add similar changes to the image tool command line in update3.md

Reply:
D - The sample test's 'wrapper' script (which calls the same command) should key off of the domain type env var (JRF or WLS).

Reply:
E - The MII user guide should have the additional text too for its mention of the image tool, and it should say something like "if your domain resource 'spec.configuration.model.domainType' is JRF, then ..."

Reply:
I agree

Reply:
I agree Tom's suggestion reads better. Fixed it.

Reply:
I am fine with doing changes in this PR for D

Reply:
I Haven't pushed it yet. I want to do D as well if it can be in the same PR. Just wondered if we try to keep doc changes and code changes separate. I will do it in the same PR since there doesn't seem to be any preference.

Reply:
Tom, can you review my changes and approve if you are satisfied. Changes to the tests will take a little longer and Monica would like to update the doc since there have been some inquiries.

Reply:
Line 265 should be removed - it's your old version of the note.